BigBaseV2/src/views/self/view_weapons.cpp

```diff
@@ -19,6 +20,18 @@ namespace big
 
 		ImGui::Checkbox("Enable Special Ammo", &g->weapons.ammo_special.toggle);
 
+		if (ImGui::Checkbox("Bypass C4 Limit", &g->weapons.bypass_c4_limit))
+		{
+			if (g->weapons.bypass_c4_limit)
+			{
+				*g_pointers->m_bypass_max_count_of_active_sticky_bombs = 99;
+			}
+			else
+			{
+				*g_pointers->m_bypass_max_count_of_active_sticky_bombs = 4;
+			}
+		}
+
```

Comment:
This could be written differently:
```cpp
if (ImGui::Checkbox("Bypass C4 Limit", &g->weapons.bypass_c4_limit))
{
    const auto limit = g->weapons.bypass_c4_limit ? 99 : 4;
    *g_pointers->m_bypass_max_count_of_active_sticky_bombs = limit;
}

if (ImGui::Checkbox("Bypass C4 Limit", &g->weapons.bypass_c4_limit))
{
    *g_pointers->m_bypass_max_count_of_active_sticky_bombs = g->weapons.bypass_c4_limit ? 99 : 4;
}
```

Compiler probably optimises it the same so it's up to you.